core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20220112_add_updated_and_transfer_job_timestamps_to_workspace_migration.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="ehigham" id="add_UPDATED_TIMESTAMP_TO_WORKSPACE_MIGRATION">
+        <addColumn tableName="V1_WORKSPACE_MIGRATION_HISTORY">
+            <column name="UPDATED"
+                    type="DATETIME"
+                    defaultValueComputed="CURRENT_TIMESTAMP"
+                    remarks="When the system scheduled migrating the WORKSPACE.">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <sql endDelimiter="~">
+            CREATE TRIGGER bump_migration_timestamp_on_update
+                BEFORE UPDATE ON V1_WORKSPACE_MIGRATION_HISTORY
+                FOR EACH ROW
+            BEGIN
+                SET NEW.UPDATED = CURRENT_TIMESTAMP();
+            END ~
+        </sql>
+    </changeSet>
+    <changeSet logicalFilePath="dummy" author="ehigham" id="add_WORKSPACE_BUCKET_TRANSFER_TIMESTAMPS">
+        <addColumn tableName="V1_WORKSPACE_MIGRATION_HISTORY">
+            <column name="WORKSPACE_BUCKET_TRANSFER_JOB_ISSUED" type="DATETIME">
+                <constraints nullable="true"/>
+            </column>
+            <column name="WORKSPACE_BUCKET_TRANSFERRED" type="DATETIME">
```

Comment:
The state of these jobs is maintained in a different table. Getting more detailed information like when the job `FINISHED` or the job outcome is a join on the migration id.
I'm using the `*_TRANSFERRED` timestamp as a positive case for advancing the migration. If it were just `JOB_END` I'd then need to do a case analysis on the outcome every time - this feels a bit clunky.
I wanted to avoid using `STARTED` because the transfer job doesn't necessarily start when you POST to google sts. Maybe not the most important detail, but that would be a nit for me lol